angular-hometask/src/app/core/breadcrumbs/breadcrumbs.component.scss

```diff
@@ -2,9 +2,39 @@
 
 :host {
```

Comment:
Good, but just assign this style to root class

Reply:
Done 093fb6e5337b9071065cad01394a1dd84974eabe